framework/Models/PostProcessors/EconomicRatio.py

```diff
@@ -173,98 +162,118 @@ def _handleInput(self, paramInput):
       @ In, paramInput, ParameterInput, the already parsed input.
       @ Out, None
     """
-    self.toDo = {}
+
+    # handle scalarVals and vectorVals from BasicStatistics
+    super()._handleInput(paramInput, childVals=self.tealVals)
+
+    # now handle tealVals
```

Comment:
what are tealVals? They don't have to come from teal right?

Reply:
is that something that Jia/Andrea put in the original postprocessor that is just a misnomer name?

Reply:
tealVals were in the original code, they contain the metrics unique to EconomicRatio. They don't have to come from teal at all. I could rename them to something like "econVals" that might be more descriptive.

Reply:
not a required change, but I do think that would be vastly more informative, if you have other changes to make as well.

Reply:
Easy find and replace task, I changed tealVals to econVals in commit 9338dde.